.ci/docker-compose-file/Makefile.local

```diff
@@ -0,0 +1,49 @@
+.PHONY: help up down ct ct-all bash run
```

Comment:
This is a helper `Makefile` to run tests in Docker Compose reusing CI Docker Compose configs. A user can have a full and easy to create test environment with all the resources.

We may opt not to keep it in this repo, though.